core/src/client/importer.rs

```diff
@@ -271,7 +269,6 @@ impl Importer {
             );
         })?;
 
-
```

Comment:
It seems that your script removed some lines unrelated to this change.
Please check it before you apply this script to CodeChain.

Reply:
I checked the scripts; there is no code to remove these kinds of lines.
I think it's your ide environment that removed these lines. Please check it.

Reply:
By the way, why not
```python
for i in range(n - 2):
```
at https://gist.github.com/junha1/3ca19ab666140b527fc9f368f785d5e9#file-use-clauses_mering_script-py-L21-L22

Reply:
@majecty timeout @sgkim126 Sorry I just checked it. I'll fix that.